GiniVision/Classes/Networking/DocumentServiceProtocol.swift

```diff
@@ -179,13 +179,13 @@ extension DocumentServiceProtocol {
                 if let error = task?.error {
                     let id = self.compositeDocument?.documentId ?? ""
                     let message = "Error sending feedback for document with id: \(id) error: \(error)"
-                    Logger.debug(message: message, event: .error)
+                    Logger.log(message: message, event: .error)
```

Comment:
What about having a function for it. So we don't always have to write `Logger.log`, which can be annoying. We could replace it with `Log(...)` or something.

Reply:
I like the idea! 💯